x-pack/lib/monitoring/outputs/elasticsearch_monitoring.rb

```diff
@@ -0,0 +1,13 @@
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License;
+# you may not use this file except in compliance with the Elastic License.
+
+module LogStash module Outputs
+  class ElasticSearchMonitoring < LogStash::Outputs::ElasticSearch
+    config_name "elasticsearch_monitoring"
+
+    def use_event_type?(client)
+      return client.maximum_seen_major_version >= 8 && params["bulk_path"] && params["bulk_path"].start_with?("/_monitoring")
```

Comment:
I don't think this logic is correct - we want to use the event type regardless of the version of Elasticsearch and only want to use it conditionally for `8.x` and above. And unless we expect/allow users to change the template to use a different bulk path (thoughts @jsvd?), we could remove the check completely and just say  
```
def use_event_type?(client)
   true
end
```

Reply:
You are right! Probably should be

```ruby
def use_event_type?(client)
   if client.maximum_seen_major_version >= 8
      params["bulk_path"] && params["bulk_path"].start_with?("/_monitoring") 
   else
     true
  end
end
```

Reply:
Yes, I think that makes sense - can we have a test to ensure correct behaviour?

Reply:
why is the conditional needed at all here? we'll always want to use the event_type because it's how the _monitoring api works, and this custom plugin won't be used for anything else

Reply:
So we can go with plain `return true` seems good to me